doc/usersguide/UnderstandingRayHistoryState.rst

```diff
@@ -1,17 +1,17 @@
-Understanding ray history state in DAG-MCNP5
-============================================
+Understanding ray history state in DAG-MCNP5/6
+==============================================
 
-Documenting calls to the track() method in MCNP5
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Documenting calls to the track() method in MCNP
```

Comment:
Can we make this the same change as in the above section header? Move to MCNP 5/6 and for extra clarity maybe we can specify that DAG-MCNP means DAG-MCNP5/6 at some point?